traefik/values.yaml

```diff
@@ -213,6 +213,18 @@ ports:
     # The port protocol (TCP/UDP)
     protocol: TCP
     # nodePort: 32443
+    # Set TLS at the entrypoint
+    # https://docs.traefik.io/v2.2/routing/entrypoints/#tls
+    tls:
+      enabled: false
+      options: ""
```

Comment:
Sorry for the long time no see. Thanks for requesting my review :-)

The options could be a yaml block (https://doc.traefik.io/traefik/https/tls/#tls-options). Can we at least mention it in the example and show it that way?

Reply:
Per other comment
> If i am not mistaken, this is to define the name of a tls options configuration, not the options themselves. If that is true this is exactly what it should be

That said, it did bring up a good point that there is no way to define a TLSOption CRD. Because of that i have added such a way.